.storybook/preview-head.html

```diff
@@ -7,5 +7,9 @@
     --mb-color-focus: #cbe2f7;
     --mb-color-bg-dark: #93a1ab;
     --mb-color-bg-light: #f9fbfc;
+    --mb-color-bg-white: #fff;
+    --mb-color-text-white: #fff;
```

Comment:
Is there a need to differentiate these colors? Wouldn't having a single `--mb-color-white` be easier to understand as both refer to pure white?

For comparison, I believe we used to have `text-brand` and now we just use `brand` directly.